### PR TITLE
SUL23-60 Style the interior page header and footer

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Styled the interior page header
- styled the library global footer

# Steps to Test
1, Review the deploy preview at https://637e5090c20c8b0008022cd6--stanford-library.netlify.app/test-basic-page-content/paragraph-components/test-text-area-content
2. verify it closely resembles the [mockups](https://www.figma.com/file/Zsz3AI81HpVF5Vnuxp3gwC/Stanford-Libraries-Design-02?node-id=2735%3A151399&t=RNkrnygPTJ4kkB0b-0).

<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/37"></a>

### CODE REVIEW

File: components/patterns/lockup.tsx [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
     <div {...props}>
       <Link href="/" className="su-no-underline">
         <div className="su-flex md:su-flex-row su-basefont-19">
-          <div><Logo /></div>
+          <Logo/>
 
           <div className="su-flex su-flex-wrap lg:su-items-center su-ml-10">
-            <div className="lg:su-flex su-items-baseline su-leading-none su-w-full lg:su-w-auto su-mt-15 lg:su-mt-0 su-logo su-text-cardinal-red su-type-4 su-mr-5 su-pr-5 lg:su-border-r su-border-solid su-border-black">
+            <div
+              className={"lg:su-flex su-items-baseline su-leading-none su-w-full lg:su-w-auto su-mt-15 lg:su-mt-0 su-logo su-type-4 su-mr-5 su-pr-5 lg:su-border-r su-border-solid su-border-black" + (whiteText ? " su-text-white" : " su-text-cardinal-red")}>
```
Comment (jdwjdwjdw) [inline components/patterns/lockup.tsx:19]: - Lockup border right is pretty hard to see in the footer. You may want to consider lightening that up a bit.
- Can you add `hover:su-text-white` if `whiteText` as well? Currently red on hover in the footer.

**Review by jdwjdwjdw — CHANGES_REQUESTED** [review]
Looking great, just a couple of minor things.

File: components/patterns/library-footer.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        <div>
+          <h2 className="su-text-m1 su-mb-20">Support Stanford Libraries</h2>
+          <div className="su-inline-block md:su-block"><DrupalLinkButton href="#">Donate</DrupalLinkButton></div>
+
+          <div className="su-flex su-mt-50 su-gap-lg su-justify-around">
+            <FooterLink href="#"><FacebookIcon/></FooterLink>
+            <FooterLink href="#"><LinkedInIcon/></FooterLink>
+            <FooterLink href="#"><TwitterIcon/></FooterLink>
+            <FooterLink href="#"><InstagramIcon className="su-fill-white" height={30}/></FooterLink>
+            <FooterLink href="#"><YoutubeIcon className="su-transition-all su-fill-white hover:su-fill-cool-grey" height={30}/></FooterLink>
```
[the remark this answers is not in the document]
Comment (pookmish) [reply components/patterns/library-footer.tsx:68]: TIL: `hocus:`

File: components/patterns/lockup.tsx (hunk 1 above)
Comment (pookmish) [reply components/patterns/lockup.tsx]: Cool call

**Pull request merged**